email-service/src/main/java/com/vgur/spring/email/services/EmailSending.java

```diff
@@ -10,30 +12,59 @@
 import org.springframework.amqp.rabbit.core.RabbitTemplate;
 import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
 import org.springframework.amqp.rabbit.listener.adapter.MessageListenerAdapter;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.mail.SimpleMailMessage;
 import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+
 @Service
 @RequiredArgsConstructor
 @Slf4j
+@EnableConfigurationProperties(
+        EmailProperties.class
+)
 public class EmailSending {
     private final JavaMailSender mailSender;
+    private final EmailProperties emailProperties;
+    private ArrayList<OrderDto> ordersToSend;
+
+    @PostConstruct
+    public void init(){
```

Comment:
why did u initialise it here ?

Reply:
I don't know where is better to initialize it